posthog/caching/tolerant_zlib_compressor.py

```diff
@@ -5,9 +5,20 @@
 from django.conf import settings
 
 import structlog
+from prometheus_client import Counter
 
 logger = structlog.get_logger(__name__)
 
+decompression_failed_counter = Counter(
```

Comment:
We've gone for capitalising elsewhere for prom metric definitions, not super important but might as well keep consistent.